webpack/assets/javascripts/react_app/components/Layout/components/__snapshots__/TaxonomyDropdown.test.js.snap

```diff
@@ -23,7 +23,7 @@ exports[`TaxonomyDropdown rendering render TaxonomyDropdown 1`] = `
     <li
       className="dropdown-header"
     >
-      Organization
+      organization
```

Comment:
This looks like an unintended change. It should still use `taxonomy.title`, right?

Reply:
This is intended, the header is shown in all caps so I didn't bother upcasing it since everywhere else that the taxonomyType is used it's lowercase